packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -3021,3 +3021,55 @@ it('should not throw on headless table', () => {
   expect(container.querySelector('.iui-table-header .iui-row')).toBeFalsy();
   expect(container.querySelector('.iui-table-body')).toBeTruthy();
 });
+
+it('should scroll to selected item in virtualized table', async () => {
+  let triggerResize: (size: DOMRectReadOnly) => void = jest.fn();
+
+  jest
+    .spyOn(HTMLElement.prototype, 'scrollTo')
+    .mockImplementation(function (this: HTMLElement, options) {
+      console.log('scroll to');
+      this.scrollTop = (options as ScrollToOptions).top ?? 0;
+      fireEvent.scroll(this, {
+        target: { scrollTop: (options as ScrollToOptions).top ?? 0 },
+      });
+    });
+  jest
+    .spyOn(UseResizeObserver, 'useResizeObserver')
+    .mockImplementation((onResize) => {
+      triggerResize = onResize;
+      return [
+        jest.fn(),
+        ({ disconnect: jest.fn() } as unknown) as ResizeObserver,
+      ];
+    });
+
+  const data = mockedData(20);
+
+  renderComponent({
+    data,
+    enableVirtualization: true,
+    getRowId: (item) => item.name,
+    scrollToItem: data[19],
```

Comment:
Minor, but I feel like we should use a different test index than the last index. If this test breaks in the future and scrolls to the last element instead of a specific index, this test would still pass. Maybe `data[9]`?